src/Initializer/time_stepping/LtsWeights/LtsWeights.cpp

```diff
@@ -337,10 +340,14 @@ LtsWeights::ComputeWiggleFactorResult
   logInfo(rank) << "The best wiggle factor is" << bestWiggleFactor << "with cost"
                 << bestCostEstimate << "and" << minAdmissibleNumberOfClusters << "time clusters";
 
-  logInfo(rank) << "Cost decreased" << (*baselineCost - bestCostEstimate) / *baselineCost * 100
-                << "% with absolute cost difference" << *baselineCost - bestCostEstimate
-                << "compared to the baseline";
-  if (baselineCost < bestCostEstimate) {
+  if (baselineCost > bestCostEstimate) {
+    logInfo(rank) << "Cost decreased" << (*baselineCost - bestCostEstimate) / *baselineCost * 100
```

Comment:
How can the cost decrease at this stage (merging?).

Reply:
When not using merging but using a wiggle factor. Then the cost can only decrease, because the baseline is the case of wiggle factor=1